test/sixpack-test.js

```diff
@@ -291,6 +292,28 @@ describe("Sixpack in node", function () {
             done();
         });
     });
+
+    it("should return error when a timeout happens in participate", function (done) {
+        nock(session.base_url).get(/\/participate/).delayConnection(1000).reply(200, {});
+        session.participate("show-bieber", ["trolled", "not-trolled"], function(err, resp) {
+            if (err) throw err;
```

Comment:
I think this statement is useless for your test purpose (and below)

Reply:
Not necessary... But since this is an async callback we should run `done(err)` instead of throwing it
```suggestion
            if (err) return done(err);
```

Reply:
actually, an err will never be thrown because of this https://github.com/sixpack/sixpack-js/blob/master/sixpack.js#L157-L160 ... I've changed to a new expectation ...